File: go/strings.go
package main

import (
	"fmt"
	"strconv"
	"strings"
	"time"
)

func main() {
	start := time.Now()
	var out []string
	for i := 0; i < 500000; i++ {
		s := strings.Repeat("luna", 16) + "-" + strconv.Itoa(i)
		out = append(out, s)
	}
	
	fmt.Println("strings", len(out))
	fmt.Printf("Time: %v\n", time.Since(start))
}

File: python/strings.py
import gc
import time
import gcstats  # noqa: F401  (registers stats at exit)

start = time.perf_counter()
out = []
for i in range(500000):
    out.append("luna" * 16 + "-" + str(i))
elapsed = time.perf_counter() - start
print("strings", len(out))
print("Time:", f"{elapsed:.3f}", "seconds")